eloquent/distribution.yaml

```diff
@@ -635,6 +636,8 @@ repositories:
       url: https://github.com/ros2-gbp/foonathan_memory_vendor-release.git
       version: 0.3.0-1
     source:
+      test_commits: false
+      test_pull_requests: true
```

Comment:
Why this combination - testing PRs but not actually the target branch? Also Dashing doesn't have `test_commits: false`.

Reply:
Removed.